File: diKoding/XamlMvvmBasic/XamlMvvmBasicVb/DataService.vb
Public NotInheritable Class DataService
    Implements IDataService

    Public Function GetPerson() As PersonData Implements IDataService.GetPerson
        ' Open Db
        ' Get data 
        ' map to PersonData class

        Return New PersonData() With { _
            .Age = 30, _
            .FirstName = "John", _
            .LastName = "Doe" _
        }
    End Function
End Class

File: diKoding/XamlMvvmBasic/XamlMvvmBasicCs/DataService.cs
namespace XamlMvvmBasic
{
    public sealed class DataService
        : IDataService
    {
        public PersonData GetPerson()
        {
            // Open Db
            // Get data 
            // map to PersonData class

            return new PersonData
                {
                    Age       = 30,
                    FirstName = @"John",
                    LastName  = @"Doe"
                };
        }
    }
}